homeassistant/components/particle/__init__.py

```diff
@@ -0,0 +1,123 @@
+"""Integration with the Particle Cloud."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_ACCESS_TOKEN, STATE_ON, STATE_UNAVAILABLE
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.config_validation import make_entity_service_schema
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+DOMAIN = "particle"
+
+_LOGGER = logging.getLogger(DOMAIN)
```

Comment:
Please use __name__ and move it above the constant `DOMAIN`:
```suggestion
_LOGGER = logging.getLogger(__name__)
```